lib/organisms/gravity-form/parsers/form.data.js

```diff
@@ -0,0 +1,93 @@
+angular
```

Comment:
Ok, but for now we should change this to a flat structure within the gravity-forms folder